### ISSUE TITLE
PR #152

### ISSUE THREAD

**Issue by limzykenneth** [earlymarket/CoPosition#154]
Hi guys,

I'm trying to merge the `landing-page` branch into `dev` and although the only actual conflict is with the Gemfile which can be solved very easily the bigger problem is with the pagination introduced in #142.

The pagination is depending on bootstrap which I removed in the `landing-page` branch because it causes conflict with materialize. I'm not sure about the implementation of `will_paginate` but I found a helper gem for it made for materialize [here](https://github.com/patricklindsay/will_paginate-materialize). Let me know how it will be best to proceed with this.

**Comment by t-harps** [comment]
Hey Kenneth, 
How long do you think it will take to get will_paginate running with materialize and for the site to all look OK? Essentially if we merge your branch into Dev we then can't merge into master until you've finished to full switch over to materialize.
@GarySiu might have a different opinion.

**Comment by limzykenneth** [comment]
I think for will paginate to work with materialize it might be as simple as just switching the gem. The whole site now looks a bit messed up but functional since I've only switched the library and did some clean up. It might be a good idea to hold off merging my branch into dev if it will be preferable to not hold up merging into master from dev. I can just occasionally merge dev into my branch and deal with conflicts there.

**Comment by t-harps** [comment]
Ok cool, I think it will be fine to merge your branch on second thoughts. Once you've solved the merge conflict i'll merge your branch into Dev.

**Comment by limzykenneth** [comment]
Done
